extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -178,8 +182,13 @@ export class AmpAnalytics extends AMP.BaseElement {
       return false;
     }
 
-    // Page was unloaded - free up owned resources.
-    this.transport_.deleteIframeTransport();
+    if (this.iniPromise_) {
```

Comment:
Hmmmm, I feel we should stop initiating iframeTransport once unlaid out, instead of wait for the promise to resolve and delete the iframe again. 
I am ok to have this as a temporary solution but longterm this should be considered together with #18231

Reply:
This can be tricky to implement, as promise cannot be canceled.

Reply:
even if we could cancel a promise, to support the following lifecycle:

```
active -> inactive : unlayoutCallback
inactive -> active : resumeCallback
```

we have to initialize everything in resumeCallback as well.